apps/meteor/tests/e2e/page-objects/admin.ts

```diff
@@ -95,4 +95,32 @@ export class Admin {
 	get btnResetRobotsFileContent(): Locator {
 		return this.page.locator('[data-qa-reset-setting-id="Robot_Instructions_File_Content"]');
 	}
+
+	get btnImportNewFile(): Locator {
+		return this.page.locator('.rcx-button--primary.rcx-button >> text="Import New File"');
+	}
+
+	get selectFileType(): Locator {
+		return this.page.locator('.rcx-select');
+	}
+
+	optionTypeFile(label: string): Locator {
+		return this.page.locator(`.rcx-option__content >> text="${label}"`);
+	}
```

Comment:
```suggestion
	async getOptionFileType(option: string): Locator {
	    await this.page.locator('.rcx-select').click();
	    return this.page.locator(`.rcx-option__content >> text="${options}"`)
	}
```

Maybe we should start combining this options/select in one getter, because we only use them together